client-app/shared/static-content/components/call-to-action-with-image.vue

```diff
@@ -30,17 +30,24 @@
 <script setup lang="ts">
 import type { RouteLocationRaw } from "vue-router";
 
-defineProps({
-  model: {
-    type: Object,
-    required: true,
-  },
+type ImagePositionType = "left" | "right";
+type ButtonType = {
+  label: string;
+  link: string;
+};
+
```

Comment:
Don't use separate types please if they are not used anywhere else, inline if enough